be/src/storage/olap_meta.cpp

```diff
@@ -51,9 +51,9 @@ const std::string META_POSTFIX = "/meta"; // NOLINT
 const std::string SECOND_POSTFIX = "_secondary";
 const size_t PREFIX_LENGTH = 4;
 
-OlapMeta::OlapMeta(std::string root_path) : _root_path(std::move(root_path)), _db(nullptr) {}
+KVStore::KVStore(std::string root_path) : _root_path(std::move(root_path)), _db(nullptr) {}
```

Comment:
You can rename `olap_meta.h/cpp` to `kv_store.h/cpp`.

Reply:
ok,thanks